modules/redshift-loader/src/main/scala/com/snowplowanalytics/snowplow/loader/redshift/Redshift.scala

```diff
@@ -212,6 +213,9 @@ object Redshift {
                       min_collector_tstamp, max_collector_tstamp,
                       compression, processor_artifact, processor_version, count_good
                       FROM ${Fragment.const0(schema)}.manifest WHERE base = $base"""
+              case Statement.AddLoadTstampColumn =>
+                sql"""ALTER TABLE ${Fragment.const0(EventsTable.withSchema(schema))}
+                      ADD COLUMN load_tstamp TIMESTAMP DEFAULT GETDATE() NULL"""
```

Comment:
Why is the `NULL` needed?  I would guess that the default ensures that `load_tstamp` is always populated.

Reply:
Old versions doesn't populate `load_tstamp`. They would throw error if it isn't made nullable.

Reply:
So this is just to enable rollback to older version in case the rollout has problems?

That's a good valid reason.  But please confirm I understand it right.

Reply:
Yep, exactly. This is for making rollback possible.